src/pickley/__init__.py

```diff
@@ -410,9 +410,6 @@ def __init__(self, given_package_spec: str, authoritative=False, settings=None):
     def __repr__(self):
         return runez.short(self.auto_upgrade_spec)
 
-    def __lt__(self, other):
```

Comment:
You don't want it to be orderable-ish any more?

Reply:
This was a left-over from a long time ago where I used to sort `PackageSpec`-s, now the package specs are:
- always in the same order as given on CLI (ie: `pickley install foo bar` -> the order `foo, bar` is respected as-is)
- finding all installed is done by sorting on name, like `[PackageSpec(x) for x in sorted(installed_names)]`

So, sorting by `PackageSpec` is not used (and the test coverage was showing that).